docs/autocomplete_compatibility.md

```diff
@@ -0,0 +1,41 @@
+# Autocomplete Compatability
+
+Many lists are used to drive autocompleting user interface components. In order to support this, a standardised concept of an "autocomplete compatible" list has been defined. A list which is autocomplete compatible has the information required to drive an autocomplete-based selection mechanism, and has that information in well-known fields so no additional configuration is required to use that list in an autocomplete mechanism that understands this standard.
+
+# Specification
+
+An autocomplete-compatible list must have:
+
+1. A `name` field, which is a string value in every record. This should be the "best" name for the identified object, such as a legal or official name.
```

Comment:
I think this is wrong to suggest that we prefer legal or official names. We should instead use the most user-focused ones?

```suggestion
1. A `name` field, which is a string value in every record. This should be the "best" name for the identified object, which should usually be the one that regular users are most likely to use - this may not always be the official or legal name. For instance, an organisation may have a well-known trading name that is different from their name as registered with Companies House.
```

Reply:
Agree with @frankieroberto